app/WebParsing/ReqParser.hs

```diff
@@ -408,15 +408,20 @@ fcesParser :: Parser Req
 fcesParser = do
     _ <- Parsec.optional completionPrefix
     fces <- creditsParser
+    dept <- Parsec.optionMaybe $ Parsec.try departmentParser
     _ <- Parsec.spaces
     _ <- fceSeparator
     _ <- Parsec.optional $ Parsec.try includingSeparator <|> Parsec.try fromSeparator
-    FCES fces <$> fcesModifiersParser
+    modifiers <- fcesModifiersParser
```

Comment:
I can move this to the `Nothing` clause but it's fine here because in #1275, I will do some pattern match with `modifiers` to append department to level

Reply:
You should move it. Code changes shouldn't be predicated on future PRs.